### PR TITLE
add episode 020-mun.md

### PR DESCRIPTION
**Pull request by alphastrata**
Show notes for episode on Mun.
Didn't have time to write the timestamps - will add later assuming the weekend is kind to me...

### CODE REVIEW

File: _episodes/020-mun.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Mun" 
+date: 2020-06-26T17:00:00+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/mun #are we naming the files ourselves or is the uploader doing it?
```
Comment (jonhoo) [inline _episodes/020-mun.md:4]: If you want a particular file name, then let me know, otherwise I'll just choose one :p

**Comment by ericseppanen** [comment]
If you want me to do a transcript, feel free to let me know the audio URL as soon the audio is the final revision.

If anyone else feels like transcribing, I can also provide a walkthrough of some of the scripts that are checked in (sorry they're not documented; they're still a work in progress).

File: _episodes/020-mun.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Mun" 
+date: 2020-06-26T17:00:00+00:00
+file: #JON to add after naming the file after upload...
```
Comment (jonhoo) [inline _episodes/020-mun.md:4]: You can just put in the URL I sent on Discord :)

File: _episodes/020-mun.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+---
+title: "Mun" 
+date: 2020-06-26T17:00:00+00:00
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e016-mun.mp3
```
Comment (jonhoo) [inline _episodes/020-mun.md:4]:
```suggestion
file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e016-mun.mp3
```

File: _episodes/020-mun.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+duration: "00:50:38"
+length: "40029157"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+First time host, long time editor Jeremy talks with Bas & Remco, creators of the Mun project. Mun is a programming language empowering creation through speedy, hot reloading iteration written in Rust. Why Rust for a project like this? that's what we explore.
```
Comment (jonhoo) [inline _episodes/020-mun.md:9]: It looks like iTunes doesn't like the `&amp;` that gets generated for the & here. Could you replace it with "and"?

Oh, and capitalize the "T" in "that's what we explore"? And maybe add "in this episode" to the end of that?

**Pull request merged**